pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -520,6 +522,15 @@ void processArguments() throws Exception {
                 functionConfig.setCustomRuntimeOptions(customRuntimeOptions);
             }
 
+            if (secretsString != null) {
+                Type type = new TypeToken<Map<String, Object>>() {}.getType();
+                Map<String, Object> secretsMap = new Gson().fromJson(secretsString, type);
+                if (secretsMap == null) {
+                    secretsMap = new HashMap<>();
```

Comment:
```suggestion
                    secretsMap = Collections.emptyMap();
```

Reply:
This map still empty, or it can add elements after?

Reply:
@Shoothzj It can be updated via `pulsar-admin functions updated` cmd if you need to.